cumulus/policies/cloudformation.py

```diff
@@ -0,0 +1,60 @@
+import awacs
+import awacs.aws
+import awacs.logs
+import awacs.iam
+import awacs.s3
+import awacs.ecr
+# TODO: Remove unnecessary awacs imports
+
+from troposphere import iam
+
+import cumulus
+
+def get_policy_cloudformation_general_access(policy_name):
+    # TODO: Return policy with permissions:
+    # 1. Full Cloudformation access to stacks prefixed with application name
+    # 2. IAM access (currently using unlimited access, but this seems like it could be limited a bit)
+    return iam.Policy(
```

Comment:
This doesn't have access to IAM in the statements anywhere..  are you just meaning with respect to the individual service.* parts?